packages/core-web/src/styles/markbind.css

```diff
@@ -333,3 +333,14 @@ h5,
 h6 {
     margin-bottom: 1rem;
 }
+
+.block-embed {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
+}
+
+.block-embed iframe {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+}
```

Comment:
We should add `top: 0` and `left: 0` to the absolute positioning, because the default might not be 0 as you thought (refer [here](https://stackoverflow.com/questions/19968979/what-are-the-default-top-left-botton-or-right-values-when-positionabsolute-is))

Reply:
ohh interesting i will add it in thanks for the tip!